workspaces/src/network/server.rs

```diff
@@ -0,0 +1,69 @@
+use std::process::Child;
+
+use portpicker::pick_unused_port;
+
+use crate::network::Sandbox;
+
+pub struct SandboxServer {
+    pub(crate) rpc_port: u16,
+    pub(crate) net_port: u16,
+    process: Option<Child>,
+}
+
+impl SandboxServer {
+    pub fn new(rpc_port: u16, net_port: u16) -> Self {
+        Self {
+            rpc_port,
+            net_port,
+            process: None,
+        }
+    }
+
+    pub fn start(&mut self) -> anyhow::Result<()> {
+        println!("Starting up sandbox at localhost:{}", self.rpc_port);
+        let home_dir = Sandbox::home_dir(self.rpc_port);
+
+        // Remove dir if it already exists:
+        let _ = std::fs::remove_dir_all(&home_dir);
+        near_sandbox_utils::init(&home_dir)?.wait()?;
+
+        let child = near_sandbox_utils::run(&home_dir, self.rpc_port, self.net_port)?;
+        println!("Started sandbox: pid={:?}", child.id());
```

Comment:
We probably want to switch these prints to some actual logging solution, but can tackle it in another issue which I will create after this review